config/Dockerfiles/ostree-boot-image/ostree-boot-image.sh

```diff
@@ -7,10 +7,10 @@ base_dir="$( pwd )"
 
 function clean_up {
     set +e
-    ara generate junit - > /home/output/logs/ansible_xunit.xml
+    ara generate junit - > ${base_dir}/logs/ansible_xunit.xml
     if [ -e "$base_dir/hosts" ]; then
         virsh screenshot --file ${base_dir}/logs/atomic-host.ppm atomic-host-fedoraah
-        ansible-playbook -i hosts ci-pipeline/playbooks/setup-libvirt-image.yml -e state=absent -e skip_init=true
+        ansible-playbook -i hosts ${base_dir}/ci-pipeline/playbooks/setup-libvirt-image.yml -e state=absent -e skip_init=true
```

Comment:
@scoheb If we can do this why did we need the dir('ci-pipeline') in the Jenkinsfile?

Reply:
@p3ck I think because I had initially gotten rid of the ci-pipeline snapshot in the container...does that makes sense?